recipes/fastparquet/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: fastparquet
+  version: "0.0.1"
+
+source:
+  fn: fastparquet-0.0.1.tar.gz
+  url: https://pypi.io/packages/source/f/fastparquet/fastparquet-0.0.1.tar.gz
+  sha256: "c17b46a6f83314434143ac583968b7b0d99f331c50fae13f82e711b64a926e2a"
+
+build:
+  skip: true  # [win]
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pandas
+    - numpy
+    - numba
+    - python-snappy
+  run:
+    - python
+    - pandas
+    - numpy
+    - numba
+    - python-snappy
```

Comment:
`python-snappy` appears to be an optional requirement. I didn't see in the code where it is actually used so I couldn't say if it made sense to leave it to the user to install if they wanted it.